build.xml

```diff
@@ -147,8 +147,10 @@
   </target>
 
   <target name="run" depends="jar"
-          description="Runs the development version of the program">
-    <java jar="${jar.dir}/${productname}.jar" fork="true"/>
+          description="Gives Instructions on how to run">
+          <echo message="To run the demonstration:"/>
+          <echo message="Build using 'ant jar'"/>
```

Comment:
There seems to be a mismatch among the description, dependencies, and echoed messages.

If "run" depends on "jar" (as specified in line 149) then why should invoking the "jar" target be part of the instructions given by the "run" target? Using "ant run" would automatically first run "jar" (if necessary) and then show the messages.